depends/patches/qt/utc_from_string_no_optimize.patch

```diff
@@ -5,7 +5,7 @@ the arm64-apple-darwin HOST.
 
 --- a/qtbase/src/corelib/itemmodels/qitemselectionmodel.cpp
 +++ b/qtbase/src/corelib/itemmodels/qitemselectionmodel.cpp
-@@ -1078,9 +1078,9 @@ void QItemSelectionModelPrivate::_q_layoutChanged(const QList<QPersistentModelIn
+@@ -1028,9 +1028,9 @@ void QItemSelectionModelPrivate::_q_layoutChanged(const QList<QPersistentModelIn
  
      if (hint != QAbstractItemModel::VerticalSortHint) {
          // sort the "new" selection, as preparation for merging
```

Comment:
Changing from `stable_sort` to `sort` affects more than just optimization, are we sure this doesn't change item ordering where not expected?

Reply:
This was introduced in b335710782c2545e6eeed67b5e1763c07eab26b0.

Related to https://github.com/bitcoin/bitcoin/pull/28880#issuecomment-1867610348.